services/expiration/expiration-notification.server.ts

```diff
@@ -60,18 +60,45 @@ export function init() {
       const { type } = job.data;
       try {
         logger.debug(`Notifications: processing job ${job.name}`);
-        let records = await getRecordsByExpiration(RecordType.DnsRecord);
-        await Promise.all(
-          records.map(async (record) => {
-            if (!record.lastNotified || record.lastNotified < dayjs().subtract(30, 'd').toDate()) {
-              await updateStatusAndNotify(type, record.id, {
-                emailAddress: record.user.email,
-                subject: `Sample ${type} expiration notice`,
-                message: `Sample ${type} expiration notice message`,
-              });
-            }
-          })
-        );
+        switch (type) {
+          case RecordType.DnsRecord:
```

Comment:
Is there a reason why we are splitting this up to dnsRecord and certificate?
TBH it feels like complexity that we don't need:
- If we want to handle them separately => we should do it in a separate queue
- If we want to handle them together => we should do all of that in a single job
I'd be OK with either, but this splitting the same worker ... I just don't see the benefit, only the added complexity

Id drop this distinction and run them both on the same queue run. Or is there something I'm missing?

Reply:
Thanks for your feedback. Agreed, it's better to process Cert/DNS notifications in the same queue without added complexity.